deploy/examples/toolbox.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
         - name: rook-ceph-tools
-          image: quay.io/ceph/ceph:v17.2.6
+          image: quay.io/ceph/ceph:v18.1.3
```

Comment:
toolbox.yaml by default should also match cluster.yaml, so for CI we'll need to replace during tests.

Reply:
done